Documentation/ApiOverview/MountPoints/Index.rst

```diff
@@ -1,174 +1,253 @@
-.. include:: ../../Includes.txt
+.. include:: /Includes.txt
+.. index:: Mount points
+.. _Mount-points:
 
+============
+Mount points
+============
 
+.. highlight:: text
 
-.. _mountpoints:
+.. contents:: This page
+   :backlinks: top
+   :local:
 
-MountPoints
+
+.. how to write *Words:*
+..    mount point, multisite, multi language support, page tree, subpage
+
+
+What are mount points?
+======================
+
+TYPO3 editors can reuse existing pages of the page tree and insert them again
+at another location without having to copy anything. To achieve this they
+create a new page and set its type to *MountPoint*. Then they select a page as
+content source. This is called *mounting*. Note that no pages nor content
+elements are copied. Instead, mounting creates a reference to content that is
+already maintained elsewhere in the installation. This also means that an
+update of that content will be shown in all mounted pages as well.
+
+Mounting may or may not include subpages, depending on what the editor chooses.
```

Comment:
?? How would you mount without subpages, that would be somewhat beside the point.

Reply:
As this comment could also refer to one of mine:  
Correct, but it's a technical option and people tend to use every technical option, sometimes in creative and unintended kind.

Reply:
But which option do you mean? I mean, if you mount a page that has subpages you don't have an option to disable the subpages or something like that. You will mount the page "and all it entails" - or did I miss something?

Reply:
That's correct.  
I referred to the option to mount a single page without subpages.

Reply:
Ah, I was confused by the word "option" and thought you meant a setting somewhere :)